### ISSUE TITLE
Will "click" keyword be totally removed from browser library

### ISSUE THREAD

**Issue by naibra** [MarketSquare/robotframework-browser#3083]
With the Click keyword be totally removed from the browser library or as stated in the User Guide, only the deprecated arguments will be removed? Can you please provide some information on this? 

Kind regards,
Namitha

**Comment by aaltat** [comment]
The Click keyword will stay, but the deprecated arguments will be removed in near future.

Because you are not the first one who asks the same question, would you have an idea how to make the document better?

@allcontributors please add @naibra for bugs.

**Comment by naibra** [comment]
I would rephrase the documentation to - 
"In the future, the arguments that are marked as *DEPRECATED* will be removed, but the Click keyword will remain available for future use."
